PayButton/paybutton-server
# Feat/411 2 chronik skeleton and get balance

Steps to test:

1. This PR should be reviewed after #417 since it requires those packages (they are included here as well so they could be reviewed simultaneously, however if there are any breaking changes there, things could get complex to merge afterwards)
2. open in you browser and see balance (example address in this url, feel free to change it): http://localhost:3000/api/address/balance/ecash:prezk74rzl4uw22wm6agcteuwztdvp5a9umns6k0d3
3. Change in constants/index.ts the ecash choice to chronik:
`export const BLOCKCHAIN_CLIENTS_CHOSEN: KeyValueT<BLOCKCHAIN_CLIENTS_OPTIONS> = {
  ecash: 'chronik',
  bitcoincash: 'grpc'
}`
4. open in you browser and see balance (now from chronik client): http://localhost:3000/api/address/balance/ecash:prezk74rzl4uw22wm6agcteuwztdvp5a9umns6k0d3

Other API endpoints will not work with chronik as they are not implemented yet, this will be addressed in future PRs.
